internal/shard/shard_manager.go

```diff
@@ -104,15 +104,16 @@ func (manager *ShardManager) GetShard(id ShardID) *ShardThread {
 	return nil
 }
 
-// RegisterIOThread registers a io-thread with all Shards present in the ShardManager.
-func (manager *ShardManager) RegisterIOThread(id string, request, processing chan *ops.StoreResponse) {
+// RegisterCommandHandler registers a command handler with all Shards present in the ShardManager.
```

Comment:
Same as earlier, what do you think of naming Manager to Registar/Registry? This name provides the precise nature of the work done by this type.

Reply:
Done.

Reply:
Let me think about this on, will discuss with you.